api/auth-handler.go

```diff
@@ -22,19 +22,20 @@ import (
 func checkRequestAuth(r *http.Request, action policy.Action) (c common.Credential, err error) {
 	// TODO:Location constraint
 	ctx := getRequestContext(r)
+	logger := ctx.Logger
 	authType := ctx.AuthType
 	switch authType {
 	case signature.AuthTypeUnknown:
-		helper.Logger.Println(5, "ErrAccessDenied: AuthTypeUnknown")
+		logger.Info("ErrAccessDenied: AuthTypeUnknown")
```

Comment:
how to choose logger.info() or helper.logger.info()?

Reply:
I think they're the same, maybe we should just stick to `helper.logger`.

Reply:
Whenever possible, use the logger extracted from a request context, because it has request ID attached.